acquisition.py

```diff
@@ -98,14 +99,14 @@ def add_device(self, driverstring, configstring):
 
         device = devs[0]
 
+        self.session.add_device(device)
+        device.open()
```

Comment:
What is the purpose of moving these two lines? This reads like change noise - a comment inserted before on why this reordering is needed and what bug it solves would be very helpful.